git_machete/client.py

```diff
@@ -1627,7 +1627,7 @@ def __pick_remote(
               "repository, or 'n' to skip this branch, or " \
               "'q' to quit the traverse: " if is_called_from_traverse \
               else f"Select number 1..{len(rems)} to specify the destination remote " \
-                   "repository, or 'q' to quit creating pull request: "
+                   "repository, or 'q' to quit creating pull request:"
```

Comment:
The space after the one before last character `:` is unnecessary and makes it difficult for tests (pep8 nad general checks will throw error about trailing whitespaces) and even if i can silence pep8 i can't do anything about general checks (well actually i can but it would involve changing general checks so that they ignore tests dir - not sure if it is a correct solution)

Reply:
Huh the space feels kinda necessary TBH here in the user-facing prompt... I'd leave it as is

Let's use `# noqa` + exclude tests from the trailing whitespace check (in fact you can exclude all Python  sources from that check, coz they're already covered by pep8 anyway)